specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json

```diff
@@ -1955,73 +1955,111 @@
         }
       }
     },
-    "/{tenantID}/oauth2PermissionGrants" : {
-      "get" : {
-       "tags": [
-         "OAuth2Permissions_get"
-       ],
+    "/{tenantID}/oauth2PermissionGrants": {
+      "get": {
+        "tags": [
+          "OAuth2Permissions_get"
+        ],
         "operationId": "OAuth2_Get",
-        "description": "Queries OAuth2 permissions for the relevant SP ObjectId of an app.",
-        "produces" : [ "application/json" ],
-        "parameters" : [
-        {
-          "name" : "$filter",
-          "in" : "query",
-          "required" : false,
-          "type" : "string",
-          "x-example" : "clientId+eq+'61ed44c3-5a1d-4639-a215-07f25129c6c3'",
-          "description": "This is the Service Principal ObjectId associated with the app"
-        },
-        {
-          "$ref": "#/parameters/ApiVersionParameter"
-        },
-        {
-          "$ref": "#/parameters/tenantIDInPath"
-        }
-       ],
-        "responses" : {
-          "200" : {
-            "description" : "OK. The operation was successful.",
-            "schema" : {
-              "$ref" : "#/definitions/Permissions"
+        "description": "Queries OAuth2 permissions grants for the relevant SP ObjectId of an app.",
+        "produces": [ "application/json" ],
+        "parameters": [
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-example": "clientId+eq+'61ed44c3-5a1d-4639-a215-07f25129c6c3",
+            "description": "This is the Service Principal ObjectId associated with the app"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/tenantIDInPath"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The operation was successful.",
+            "schema": {
+              "$ref": "#/definitions/PermissionsListResult"
```

Comment:
So, if I understand correctly, it was a list operation and was a mistake? If that's the case:
- Rename it OAuth2_Get to OAuth2_List
- Paging operations requires a special second declarations at the end of the file for GraphRbac. Example:
https://github.com/Azure/azure-rest-api-specs/blob/6a18208aed692f02215a2ea50736c0a8d5ac7329/specification/graphrbac/data-plane/Microsoft.GraphRbac/stable/1.6/graphrbac.json#L2075-L2117

Reply:
Let us get it right.
@psignoret, it is clear the `oauth2PermissionGrant` GET returns a list, and the question here is whether or not it is a pagable list.

Reply:
Actually no, the question is not about "is this pageable", because the Autorest option is poorly named about that ;(. The question is "is this a list", if yes, we need to use the pageable options.
However, if we know for sure there is no second page, we might be able to skip this second operations definitions

Reply:
It is a list, because the payload tells me it is a list, see below. Being pagable or not turns out not a concern, because the pertinent scenario is to query the grant made on a service principal; and what I found out is you can't grant more than once on a service principal, so literally you have a list with only one entry :). I would like to get confirm from @psignoret, but I would not worry about that much by not having a paging right now.
```
{  
   "odata.metadata":"https://graph.windows.net/00000000-0000-0000-0000-000000000000/$metadata#oauth2PermissionGrants",
   "value":[  
      {  
         "clientId":"75c658dd-7ff1-41d5-a4e4-35c2d9c72432",
         "consentType":"AllPrincipals",
         "expiryTime":"2019-10-29T17:58:22.92475",
         "objectId":"3VjGdfF_1UGk5DXC2cckMu9Vovv3CJtPtnMxj8uH8EI",
         "principalId":null,
         "resourceId":"fba255ef-08f7-4f9b-b673-318fcb87f042",
         "scope":"user_impersonation",
         "startTime":"2018-10-29T17:58:22.92475"
      }
   ]
}
```

Reply:
Okay, it is a list and page-able. I created 200 service principal and confirmed this.
I have addressed all the comments. please take another look.